pkg/warden/reporter/report.go

```diff
@@ -112,9 +112,19 @@ func (r *Reporter) do(info scout.WardenInfo) (*http.Response, error) {
 
 	reader := bytes.NewReader(data)
 
-	url := fmt.Sprintf("https://%s%s", r.PivotCubeHost, "/api/v1/cube/scout/heartbeat")
+	// default, use https as scheme
+	uri, err := url.Parse(r.PivotCubeHost)
+	if err != nil {
+		return nil, err
+	}
+
+	uri.Path = "api/v1/cube/scout/heartbeat"
+	if len(uri.Host) == 0 {
+		uri.Scheme = "https"
+		uri.Host = r.PivotCubeHost
+	}
 
-	resp, err := r.Client.Post(url, "application/json", reader)
+	resp, err := r.Client.Post(uri.String(), "application/json", reader)
```

Comment:
Not sure if we should delete `&tls.Config{InsecureSkipVerify: true}` when it use http scheme, It doesn't matter?

Reply:
no problem